lib/module.js

```diff
@@ -51,6 +51,7 @@ module.exports = async function gtmModule (_options) {
   script = `if(!window._gtm_init){window._gtm_init=1;${script}}`
 
   // Add google tag manager <script> to head
+  this.options.head = typeof this.options.head === 'function' ? {} : this.options.head
```

Comment:
This silently removes user-provided head function. Meanwhile working on a nuxt enhancement to allow providing a custom head in a better way (`app/head.js`) maybe adding a warning like `[@nuxtjs/gtm] head is provided as a function which is not supported by this module at the moment. Removing user-provided head.` (or something like this)

Reply:
okay i found it. so i try to fix this right way that support.

Reply:
i think that your better way is good alternative. so i'm gonna remove to my head function in nuxt.config.js. Inadvertently i made you confused, but i want to improve to about this.

thanks.

Reply:
No i mean this PR idea is awesome because users should be aware of this current incompatibility. If you add the warning, would be happy to merge it for now :)

Reply:
okay just moment :)